Automattic/wp-calypso
# DO NOT MERGE: Update PopoverMenuItem to accept a Material icon in lieu of a gridicon

#### Changes proposed in this Pull Request

* Allow `PopoverMenuItem` to use a Material icon rather than a Gridicon, and update the Edit and Stats icons for the Posts ellipsis menu with Material icons.

**Before**

<img width="253" alt="Screen Shot 2020-07-29 at 5 19 33 PM" src="https://user-images.githubusercontent.com/2124984/88854743-bee72700-d1bf-11ea-8232-32e04a910b6a.png">

**After**

<img width="301" alt="Screen Shot 2020-07-29 at 5 19 18 PM" src="https://user-images.githubusercontent.com/2124984/88854755-c4dd0800-d1bf-11ea-989c-94d3b75f026f.png">

#### Testing instructions

* Switch to this PR
* Navigate to /posts and click the ellipsis icon for a single post
* The icons for Edit and Stats should show Material icons (outlined style) rather than Gridicons
